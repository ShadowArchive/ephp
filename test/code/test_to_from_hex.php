<?php

var_dump(bin2hex(hex2bin("A")));
var_dump(bin2hex(hex2bin("AB")));

