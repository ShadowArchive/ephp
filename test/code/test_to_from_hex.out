
Warning: hex2bin(): Hexadecimal input string must have an even length in {{CWD}}/test/code/test_to_from_hex.php on line 3
string(0) ""
string(2) "ab"
